File: UberT1Costumer/Models/Request.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace UberT1Costumer.Models
{
    public class Request
    {
        public string Type { get; set; }
        public Object Body { get; set; }
        public string RequestEntity = "costumer";
    }
}

File: src/main/java/models/Request.java
package models;

public class Request
{

    private String type;
    private Object body;
    private String RequestEntity;

    public Request(String type, Object body, String requestEntity)
    {
        this.type = type;
        this.body = body;
        this.RequestEntity = requestEntity;
    }

    public void setType(String type)
    {
        this.type = type;
    }

    public void setBody(Object body)
    {
        this.body = body;
    }

    public String getRequestEntity()
    {
        return RequestEntity;
    }
    public String getType()
    {
        return type;
    }

    public Object getBody()
    {
        return body;
    }

    public String toString()
    {
        return "Type: " + type + " Object: " + body.toString() + " RequestEntity: " + RequestEntity;
    }
}
